avionics/envs/board/include/sensors/GPS.h

```diff
@@ -71,6 +71,7 @@ PLEASE READ ME!
 /*Includes------------------------------------------------------------*/
 #include <HAL/port_impl.h>
 #include <TinyGPS.h>
+#include <array>
```

Comment:
It's used later in the file; I suppose <array> must've used to be part of HAL/port_impl.h or something

Reply:
Wait so which change requires this? I feel like if the change wasn't in GPS that indicates that this include should be placed elsewhere / where it is needed.

Reply:
In either case it'd be poor practice to leave it out - std::array is used in this header file.
It might've been something from the radio header which included <array> but I'm not sure. But this is where the header is needed, and it's not immediately clear where else.

Reply:
Ah okay I see. Yeah as long as its used here.